### PR TITLE
Changed labels were not saved in `.C` file

### PR DESCRIPTION
**Pull request by couet**
The changed labels were not saved in `.C` macros generated with `SaveAs`. This was pointed out here:
https://github.com/root-project/root/issues/13122

### CODE REVIEW

File: hist/hist/src/TF2.cxx [hunk 1]
```diff
@@ -842,35 +842,40 @@ void TF2::Save(Double_t xmin, Double_t xmax, Double_t ymin, Double_t ymax, Doubl
...
-   SaveFillAttributes(out, GetName(), 0, 1001);
-   SaveMarkerAttributes(out, GetName(), 1, 1, 1);
-   SaveLineAttributes(out, GetName(), 1, 1, 4);
+   SaveFillAttributes(out, f2Name.Data(), 0, 1001);
+   SaveMarkerAttributes(out, f2Name.Data(), 1, 1, 1);
+   SaveLineAttributes(out, f2Name.Data(), 1, 1, 4);
 
    if (GetNpx() != 100)
-      out<<"   "<<GetName()<<"->SetNpx("<<GetNpx()<<");"<<std::endl;
+      out<<"   "<<f2Name.Data()<<"->SetNpx("<<GetNpx()<<");"<<std::endl;
```
Comment (linev) [inline hist/hist/src/TF2.cxx:863]: Why only custom `npx` is preserved? I guess, `npy` is missing here.

**Review by linev — CHANGES_REQUESTED** [review]
One need to add missing `np` attributes for `TF2` and `TF3`

File: hist/hist/src/TF2.cxx (hunk 1 above)
Comment (couet) [reply hist/hist/src/TF2.cxx:863]: I'll look .. I do not change that code... I was like that.

**Comment by couet** [comment]
The last push fixes the points you mentioned. Thanks!

**Review by linev — APPROVED** [review]

**Pull request merged**